src/types.ts

```diff
@@ -20,3 +21,77 @@ export interface RequestOptions<Payload extends RequestPayload, Response> {
   payload: Payload;
   getProvider?: () => Promise<BitcoinProvider | undefined>;
 }
+
+// RPC Request and Response types
+
+export type RpcId = string | number | null | undefined;
+
+export interface RpcBase {
+  jsonrpc: '2.0';
+  id: RpcId;
+}
+export interface RpcRequest<T extends string, U> extends RpcBase {
+  method: T;
+  params: U;
+}
```

Comment:
This doesn't seem to be used.

Reply:
it's used by the wallets but you are right it should be removed 👍